File: Version Alpha 1.1/losses_backend/services/data-service/src/services/tanks.service.ts
import Tank from "../models/tanks.model";


export class TankService
{

    public static async getTanks()
    {
        try {
            
            const data = await Tank.findAll(
                {
                    attributes:['id','name',['isolated','isIsolated'],['d','diameter'],['hro','spaceTank'],['alfa','alpha'],['hs','tankHeight'],['hnx','hlx'],['hnn','hln'], 'pr', 'ge','tc','hro','hs']
                }
            )
            return data;

        } catch (error) {
            throw new Error('No se encontraron datos');
        }
    }
}

File: Version Alpha 1.1/losses_backend/services/preprocesing-service/services/tanksService.py
from dto.tankDTO import TankDTO
from utils.apiObject import APIObject

class TankService:

    def __init__(self):
        
        self.data = []        
        self.api = APIObject('http://losses_data_service:5000/api/v1/losses-data/get-tanks')

    def getTanks(self):
        
        response = self.api.getData()
        for row in response:    
            self.data.append(TankDTO(
                row['id'], 
                row['name'], 
                row['isIsolated'], 
                row["diameter"], 
                row["spaceTank"],
                row["alpha"],
                row["tankHeight"],
                row["hlx"],
                row["hln"],
                row["pr"],
                row["ge"],
                row["tc"],
                row["hro"],
                row["hs"] 
                ))

        return self.data